css/comp-page.css

```diff
@@ -1,48 +1,64 @@
-@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
-
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap");
 :root {
-    --primary-colour: #93dacd;
-    --secondary-colour: #e2e3e5;
+    --primary-colour: #00b594;
+    --secondary-colour: #d0f7f0;
+    --primary-font-colour: #fff;
+    --secondary-font-colour: #000;
+    --font: "Montserrat", sans-serif;
 }
 
 * {
     box-sizing: border-box;
     font-family: "Montserrat", sans-serif;
+    font-weight: 400;
 }
 
 .site-nav {
     position: fixed;
+    top: 0;
+    bottom: 0;
     margin-left: 5em;
-    margin-top: 0.5em;
+    margin-top: 7em;
+    overflow-y: auto;
 }
 
-.nav-heading {
-    font-size: 1.07rem;
+.nav-main-heading {
+    font-size: 1rem;
     padding: 0.2em 0.5em;
     border-radius: 0.3em;
     display: block;
     width: 100%;
+    color: #626364;
+    font-weight: 500;
+    margin-top: 0.6em;
 }
 
-.nav-heading:hover {
+.nav-sub-heading {
+    font-size: 1.07rem;
+    padding: 0.2em 0.5em;
+    border-radius: 0.3em;
+    display: block;
+    width: 93%;
+    color: #585a5c;
+    margin-top: 0.3em;
+    margin-left: 0.5em;
+}
+
+.nav-sub-heading:hover {
     background-color: #d6f3ee;
 }
 
-.nav-heading > a {
+.nav-sub-heading > a {
     display: block;
     width: 100%;
     text-decoration: none;
     color: #585a5c;
```

Comment:
this color is getting reused on line #82 so put all reused colors in a variable.

Another good practice which helps in extending your lib/debugging is to use variables for colors and z-indices.